internal/cmd/root.go

```diff
@@ -24,8 +25,12 @@ func NewRootCmd() *cobra.Command {
 		Short:        "A cli for executing SQL statements on a libSQL or SQLite database",
 		Args:         cobra.MatchAll(cobra.ExactArgs(1), cobra.OnlyValidArgs),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			dbUri := args[0]
+			if rootArgs.authToken != "" {
+				dbUri = fmt.Sprintf("%s?authToken=%s", dbUri, rootArgs.authToken)
+			}
```

Comment:
Another thing that I just realize that I wasn't clear enough while writing the issue. It would be better to have `Auth` as a `ShellConfig` field. This way this logic will be hidden from shell users (like `turso-cli`)